esp/GBPlay/main/build/CMakeFiles/3.22.3/CMakeCCompiler.cmake

```diff
@@ -0,0 +1,72 @@
+set(CMAKE_C_COMPILER "/usr/bin/cc")
```

Comment:
I took a look and we have `esp/GBPlay/build/` in our `.gitignore`, so I'm not sure how these got included. Can you remove everything under `build/` and rebase?

Reply:
yeah I can give that a go, might've been something I did